addons/spectrum/XEH_preInit.sqf

```diff
@@ -192,4 +192,60 @@ GVAR(voiceLinePacks) set ["british", 			[GVAR(voiceLinesBritishList), GVAR(voice
 GVAR(voiceLinePacks) set ["alienElectronic", 	[GVAR(voiceLinesAlienEletronicList), GVAR(voiceLinesAlienElectronicWeights), "Alien Electronic"]];
-GVAR(voiceLinePacks) set ["police", 			[GVAR(voiceLinesPoliceList), GVAR(voiceLinesPoliceWeights), "Police Radio"]];
\ No newline at end of file
+GVAR(voiceLinePacks) set ["police", 			[GVAR(voiceLinesPoliceList), GVAR(voiceLinesPoliceWeights), "Police Radio"]];
+
+
+// Spectrum autoline settings
+[
+    QGVAR(spectrumAutoline), // Internal setting name, should always contain a tag! This will be the global variable which takes the value of the setting.
+    "CHECKBOX", // setting type
+    ["Spectrum Autoline", "Allow players to automatically draw map lines from their position in the direction they're facing, when using the Spectrum Device"],
+    ["Crows Electronic Warfare", "Spectrum Autoline"],
+    true,
+    nil
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(spectrumAutolineColor1), // Internal setting name, should always contain a tag! This will be the global variable which takes the value of the setting.
+    "EDITBOX", // setting type
+    ["Spectrum Autoline Colour 1", "Colour that should be associated with the autoline hotkey. Must be in CfgMarkerColors"],
+    ["Crows Electronic Warfare", "Spectrum Autoline"],
+    "ColorRed",
+    nil
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(spectrumAutolineColor2), // Internal setting name, should always contain a tag! This will be the global variable which takes the value of the setting.
+    "EDITBOX", // setting type
+    ["Spectrum Autoline Colour 2", "Colour that should be associated with the autoline hotkey. Must be in CfgMarkerColors"],
+    ["Crows Electronic Warfare", "Spectrum Autoline"],
+    "ColorGreen",
+    nil
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(spectrumAutolineColor3), // Internal setting name, should always contain a tag! This will be the global variable which takes the value of the setting.
+    "EDITBOX", // setting type
+    ["Spectrum Autoline Colour 3", "Colour that should be associated with the autoline hotkey. Must be in CfgMarkerColors"],
+    ["Crows Electronic Warfare", "Spectrum Autoline"],
+    "ColorBlue",
+    nil
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(spectrumAutolineColor4), // Internal setting name, should always contain a tag! This will be the global variable which takes the value of the setting.
+    "EDITBOX", // setting type
+    ["Spectrum Autoline Colour 4", "Colour that should be associated with the autoline hotkey. Must be in CfgMarkerColors"],
+    ["Crows Electronic Warfare", "Spectrum Autoline"],
+    "ColorYellow",
+    nil
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(spectrumAutolineNoise), // Internal setting name, should always contain a tag! This will be the global variable which takes the value of the setting.
+    "SLIDER", // setting type
+    ["Spectrum Autoline Noise", "How accurate should the start point of the autoline be (m)"], 
```

Comment:
Wording might be ambigious. Do people who don't know much about it know if 0m means dead accurate, or if 100 means 100% accurate? 

It is somewhat intuitive for us, but you could possible clarify by writing:  
"_how much the start position on the map can differ in meters to your actual position_", or just include:  
"_0 means absolute accurate, 100 means it can be 100m off the actual position_".

Reply:
Yeah, good call. Have also renamed from "Noise" to "Deviation" for additional clarity (and to avoid confusion with `Add Sound` etc.)